enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -36,8 +49,8 @@ export function useLoadQuestion({
 
   // Keep track of the latest question and query results.
   // They can be updated from the below actions.
-  const [result, setQuestionResult] = useState<SdkQuestionResult>({});
-  const { question, queryResults } = result;
+  const [questionState, setQuestionState] = useReducer(questionReducer, {});
```

Comment:
This allows us to update the `question` without modifying the `queryResults` and vice-versa.